NebulaModel/Packets/Combat/CombatStatFullHptPacket.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NebulaModel.Packets.Combat;
+
+public class CombatStatFullHptPacket
```

Comment:
Should this be `CombatStatFullHpPacket` or is the `t` intentional?

Reply:
oops, this is a typo